clippy_lints/src/operators/arithmetic_side_effects.rs

```diff
@@ -184,6 +184,34 @@ impl ArithmeticSideEffects {
         }
     }
 
+    /// There are some integer methods like `wrapping_div` that will panic depending on the
+    /// provided input.
+    fn manage_method_call<'tcx>(
+        &mut self,
+        args: &[hir::Expr<'tcx>],
+        cx: &LateContext<'tcx>,
+        instance: &hir::Expr<'tcx>,
+        ps: &hir::PathSegment<'tcx>,
+    ) {
+        const METHODS: &[&str] = &["wrapping_div", "wrapping_rem", "wrapping_rem_euclid"];
```

Comment:
I will open a PR in the rust repo to create symbols for these strings

Reply:
On a second thought, are the Rust devs going to allow the inclusion of more symbols for Clippy?